### PR TITLE
RP-713 remove California from instance wide defaults

### PR DESCRIPTION
**Pull request by ftdholbrook**

### CODE REVIEW

File: webapp/src/main/resources/application.yml [hunk 1]
```diff
@@ -36,9 +36,6 @@ reporting:
   landing-page-url: forward:/landing.html
   min-item-data-year: 2016
   school-year: 2019
-  state:
-    code: CA
-    name: California
```
Comment (esotrope) [inline webapp/src/main/resources/application.yml]: nothing needs to be added to `-test.yml` after these are removed?

**Comment by ftdholbrook** [comment]
Smoke tested by creating a new tenant locally, and ran IT tests

File: webapp/src/main/resources/application.yml [hunk 2]
```diff
@@ -36,9 +36,6 @@ reporting:
   landing-page-url: forward:/landing.html
   min-item-data-year: 2016
   school-year: 2019
```
Comment (esotrope) [inline webapp/src/main/resources/application.yml]: looking at these again - does it really make sense to have defaults for these:
`min-item-data-year`
`school-year`
?

Comment (ftdholbrook) [reply webapp/src/main/resources/application.yml]: I wondered about that too, but if we remove them, then they have to become required for tenants correct?

File: webapp/src/main/resources/application.yml (hunk 1 above)
Comment (ftdholbrook) [reply webapp/src/main/resources/application.yml]: looks to me like they are already there in all the cases I could find.

File: webapp/src/main/resources/application.yml (hunk 2 above)
Comment (esotrope) [reply webapp/src/main/resources/application.yml]: sure would - but i think rightfully so - the 2016 is an SBAC item specific thing and school-year _should_ default to the current year someone is making the tenant for

File: webapp/src/main/resources/application.yml (hunk 1 above)
Comment (esotrope) [reply webapp/src/main/resources/application.yml]: kew

File: webapp/src/main/resources/application.yml (hunk 2 above)
Comment (ftdholbrook) [reply webapp/src/main/resources/application.yml]: Want to open a ticket for that @esotrope?

Comment (esotrope) [reply webapp/src/main/resources/application.yml]: sure - first i want @malaffoon 's buy in though - thoughts Mark?

Comment (malaffoon) [reply webapp/src/main/resources/application.yml]: min-item-data-year should be left alone; yeah, it is specific to the migrated legacy data for SBAC CA deployment but having an "old" value as the default is very safe.

school-year should be managed explicitly. I suppose we could remove the default and make them set it by tenant.

**Comment by malaffoon** [comment]
Btw @ftdholbrook , i'm waiting for the follow up change to RDW_Ingest?

**Comment by ftdholbrook** [comment]
> Btw @ftdholbrook , i'm waiting for the follow up change to RDW_Ingest?

kind of didn't realize there were any.  I'll go take a look.

**Pull request merged**